resources/warnings.json

```diff
@@ -6251,8 +6251,8 @@
     "url": "https://jenkins.io/security/advisory/2019-10-23/#SECURITY-1003",
     "versions": [
       {
-        "lastVersion": "2.3",
-        "pattern": ".*"
+        "lastVersion": "2.4.4",
+        "pattern": "(1[.]|2[.][0-3][.]|2[.]4[.][0-4]).*"
```

Comment:
Excludes `2.4`. No need to mess with this yourself, @Wadeck can fix up the regex, we have a tool to do that.

Reply:
Sorry but I don't understand what action I should take

Reply:
Ok :)

Reply:
Looking into this 👀. I think it was corrected in [353.v20e9cff705d1](https://github.com/jenkinsci/sonar-gerrit-plugin/releases/tag/353.v20e9cff705d1)

Reply:
> Excludes 2.4

There is no such version in https://repo.jenkins-ci.org/ui/repos/tree/General/releases/org/jenkins-ci/plugins ;)

Reply:
No, I can tell you it was fixed in 2.4.5 :)

Reply:
https://github.com/jenkinsci/sonar-gerrit-plugin/commit/d86de84131660051de7a90195478761b7d087630

Reply:
You're too reactive, I saw my mistake in the meantime :D

Reply:
> There is no such version

Good point, I checked Git tags only.

Reply:
> No, I can tell you it was fixed in 2.4.5 :)

Right, my point is that 2.4 isn't matching the regex of versions with the vulnerability.

Reply:
the 2.4.5 point was for me, because I first thought it was done a bit later with the credentials usage, but actually it was fixed in 2.4.5 with the Secret usage as first approach ^^